e2e/backend.e2e.spec.ts

```diff
@@ -21,7 +21,13 @@ test.describe.serial('KubernetesBackend', () => {
   let page: Page;
 
   test.beforeAll(async({ colorScheme }, testInfo) => {
-    [electronApp, page] = await startSlowerDesktop(testInfo);
+    [electronApp, page] = await startSlowerDesktop(testInfo, {
+      virtualMachine: {
+        mount: {
+          type: MountType.REVERSE_SSHFS,
```

Comment:
I don't understand why this is necessary?

Reply:
Because macOS defaults to virtiofs, and we later flip VM type to qemu, which fails because that isn't compatible.